nautobot/core/templates/inc/custom_fields/panel_data.html

```diff
@@ -15,6 +15,15 @@
                             <a href="{{ value }}">{{ value|truncatechars:70 }}</a>
                         {% elif field.type == 'multi-select' and value %}
                             {{ value|join:", " }}
+                        {% elif field.type == 'json' and value %}
```

Comment:
Wonder if this should be more specifically `and value is not None` - might be desirable to render `{}`, `[]`, `""` as JSON?